src/package.json

```diff
@@ -19,21 +19,26 @@
     "node": ">=14"
   },
   "dependencies": {
+    "@cosmjs/launchpad": "^0.24.0-alpha.25",
+    "@cosmjs/proto-signing": "^0.24.0-alpha.25",
```

Comment:
how large is this one? I think we can encode it in js side and use native wallet core `signDirect` method

Reply:
will check that 👍